packages/instrumenter/src/transformers/babel-transformer.ts

```diff
@@ -22,6 +22,18 @@ export const transformBabel: AstTransformer<AstFormat.JS | AstFormat.TS> = ({ ro
         // Don't mutate type declarations or import statements
         path.skip();
       } else {
+        if (
+          options.specificMutants?.length &&
```

Comment:
We could optimize performance a bit here by `skip`-ing parts of the AST that fall outside of the mutant range. But that is definitely a nice-to-have

Reply:
Great idea, but I'm not sure how I'd go about writing a test for that since the observable output would still the same with or without it